E-Commerce/readme.md

```diff
@@ -46,13 +46,100 @@ erDiagram
 Write SQL queries to complete the following tasks:
 
 - [ ] List all the products whose name contains the word "socks"
+SELECT * 
+FROM products WHERE product_name 
+LIKE '%socks%';
+
 - [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+SELECT * 
+FROM 
+    product_availability 
+JOIN  
+    products ON product_availability.prod_id = products.id 
+WHERE 
+    unit_price > 100;
+
 - [ ] List the 5 most expensive products
+SELECT prod_id, product_name, unit_price
+FROM
+    product_availability
+JOIN
+    products ON product_availability.prod_id = products.id
+ORDER BY
+    product_availability.unit_price DESC
+LIMIT 5;
+
 - [ ] List all the products sold by suppliers based in the United Kingdom. The result should only contain the columns product_name and supplier_name
+SELECT 
+    product_name, 
+    supplier_name 
+FROM 
+    suppliers 
+JOIN 
+    product_availability ON suppliers.id = product_availability.supp_id 
+JOIN 
+    products ON product_availability.prod_id = products.id 
+WHERE 
+    suppliers.country = 'United Kingdom';
+
 - [ ] List all orders, including order items, from customer named Hope Crosby
+SELECT
+    order_id,
+    order_date,
+    order_reference,
+    customers.name,
+    product_id,
+    supplier_id,
+    quantity
+FROM
+    orders
+JOIN
+    customers ON orders.customer_id = customers.id
+JOIN
+    order_items ON orders.id = order_items.order_id
+WHERE
+    customers.name = 'Hope Crosby';
+
 - [ ] List all the products in the order ORD006. The result should only contain the columns product_name, unit_price, and quantity
+SELECT
+    product_name,
+    unit_price,
+    quantity
+FROM
+    orders
+JOIN
+    order_items ON orders.id = order_items.order_id
+JOIN
+    product_availability ON order_items.product_id = product_availability.prod_id
```

Comment:
I see the same product multiple times in the result list. Have another look at the join with `product_availability`. How do we figure out which row in `product_availability `matches an order item? Does it only depend on the product id, or is there another important column involved?